docs/api-and-integration/metadata/github.md

```diff
@@ -3,32 +3,51 @@ title: GitHub Issue Tracking Integration
 
-## Settings
+![Octopus release with GitHub issues](octo-github-release-details.png "width=500")
 
-The GitHub extension only requires the base Url configuration value, which defaults to https://github.org.
+![Octopus deployment with generated release notes](octo-github-release-notes.png "width=500")
 
-This is required when resolving issue references that cross repo boundaries. For example, you might have a commit message with the following content:
+This page described how to configure this functionality for Octopus and Jira.
 
-```
-Fix bug with X
+## Connecting GitHub and Octopus Deploy
 
-Resolves MyOrg/SomeOtherRepo#1234
-```
+The GitHub Issue Tracker extension is very easy to configure with a small number of settings.
 
-`MyOrg/SomeOtherRepo#1234` refers to issue \#1234 in the `SomeOtherRepo` repository belonging to the `MyOrg` organization. While not all that common, this syntax is used when issues are tracked in a separate repo to the commit that resolves the issue.
+1. Configure the GitHub extension.
 
-### Username/Password
+    In the Octopus web portal, navigate to **{{Configuration,Settings,GitHub Issue Tracker}}** and set the
+    **GitHub Base URL**. This is required when resolving issue references that cross repo boundaries. For example, you might have a commit message with the following content:
 
-If you specify a username and password/personal access token then Octopus will retrieve issue descriptions when viewing package details and when creating releases. 
+    ```
+    Fix bug with X
 
-### Release Note Prefix
+    Resolves MyOrg/SomeOtherRepo#1234
+    ```
 
-Once you've specified a username/password, if you also specify a release note prefix the Octopus extension will look through the issue comments for one that starts with that prefix. If it finds one it will use the text following the prefix as the `WorkItemLink.Description`. If you leave this field blank, or a comment starting with the prefix isn't found, the issue's title will be used for the `WorkItemLink.Description`.
+    `MyOrg/SomeOtherRepo#1234` refers to issue \#1234 in the `SomeOtherRepo` repository belonging to the `MyOrg` organization. While not all that common, this syntax is used when issues are tracked in a separate repo to the commit that resolves the issue.
+
+    Ensure the **Is Enabled** property is set as well.
+
+2. Configure the Release Note Options (optional).
+
+    - **Username/password**: Set these values to allow Octopus to connect to GitHub and retrieve issue (work item) details from _private repositories_ when viewing packages or creating releases. If these are not provided, just the raw work item references will be used as the work item link descriptions. If they are provided the work item's title will be used as the work item link's description.
+
+    The password should be an personal access token, rather than an actual password. You can create a token in your GitHub account settings in the 'Developer settings' area.
```

Comment:
```suggestion
    The password should be a personal access token, rather than an actual password. You can create a token in your GitHub account settings in the 'Developer settings' area.
```